modules/olm-certificates.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificate-types-descriptions.adoc
+
+[id="olm-certificates_{context}"]
+= OLM certificates
```

Comment:
I think this is a good summary of what @ecordell and @awgreene have described.  

If there are no objections to the wording I'm lgtm on this :+1: